RestaurentManagement/src/main/java/com/restapi/services/BillService.java

```diff
@@ -0,0 +1,134 @@
+package com.restapi.services;
+
+import com.restapi.constants.RMConstant;
+import com.restapi.dto.BillDTO;
+import com.restapi.exceptions.RMValidateException;
+import com.restapi.mapper.BillMapper;
+import com.restapi.models.Bill;
+import com.restapi.models.Menu;
+import com.restapi.repositories.BillRepository;
+import com.restapi.repositories.MenuRepository;
+import com.restapi.response.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import java.util.*;
+
+@Service
+public class BillService {
+
+    private final BillRepository billRepository;
+    private final MenuRepository menuRepository;
```

Comment:
Shouldn't call another repo if it doesn't relate to this service. Pls call menuService and create some functions if this service needed